Documentation/CodingGuidelines

```diff
@@ -586,15 +585,13 @@ Writing Documentation:
       versions.
 
   - If you still need to refer to an example person that is
-    third-person singular, you may resort to "singular they" to avoid
+    third-person singular, you may resort to singular "they" to avoid
     "he/she/him/her", e.g.
 
       A contributor asks their upstream to pull from them.
 
     Note that this sounds ungrammatical and unnatural to those who
-    learned that "they" is only used for third-person plural, e.g.
-    those who learn English as a second language in some parts of the
-    world.
+    learned that "they" is only used for third-person plural.
```

Comment:
Those who wrote the original felt the need to do so, and the claim "there is no need" needs to be justified.

It is helpful to have such an explanation because some readers are not even aware that language is taught differently in different parts of the world.

Reply:
Many native English speakers also believe that "they" is only a third person plural. That's not restricted to non-native English speakers.

Also, it's weird to give an example for this. Either a person learned that "they" can be both singular and plural or they didn't. Whether they learned English as a second or other language is irrelevant.